src/faq.md

```diff
@@ -435,19 +436,15 @@ staying within smi range is still good practice.
 
 ### Q. How are typed lists handled when compiled to JavaScript?
 
-JavaScript offers typed arrays
-that are compatible with Dart’s typed lists.
-The mapping is trivial—for example,
-Float32List becomes a Float32Array.
-The one exception today is that dart2js does not support 64-bit integers
-and thus does not support Int64List or Uint64List.
-Dart code compiled via dart2js results in a runtime exception
-if either of those lists is used.
+JavaScript offers typed arrays compatible with Dart’s typed lists.
+The mapping is trivial: `Float32List` becomes a `Float32Array`.
+One exception exists: the production web compiler does not support
+64-bit integers: `Int64List` or `Uint64List`. Compiling Dart code with
+either of those lists results in a runtime exception.
```

Comment:
Just FYI, [semantic line breaks](https://sembr.org/) like these are a convention the Dart and Flutter sites often use

Reply:
Just learned what this is. Great point. I'll fix some here and others later.